Arachne Fixes

JFerrix opened the pull request:
# Description

This is primarily targeted to Arachne but also future taur species or potential character modifications that use the tauric inventory preset.
Generally speaking arachne players have suffered from a very weird and wonky inventory that has not only been a sharp and unlogical deviation from the standart inventories but also had significant effects on gameplay as several clothing items. Especially space capable suits were not able to be worn, making Arachne substantially more susceptible against any forms of spacing or depressurisation as they simply dont have the means as other species to protect themselves due to this inventory.
This patch is a simple and basic fix and will make clothing look substantially worse as now pants are visible but it fixes these core issues.

The displacement map changes lowest layer clothing to remove legs and potentially weird textures.

Further planned changes will involve a respriting of clothing to work with Arachne characters alongside potential bugfixes. Due to the amount of hard- and softsuits in the game it was not tested which work or dont work. Conceptually all should work but this is not a guaranteed confirmation

Update: Due to the game really not wanting to play by the rules I will attempt to program a system to manually reassign sprites to species.

---

# TODO

- [x] Fix Inventory
- [x] Fix Age
- [x] Implement Displacement map for jumpsuits
- [ ] Resprite clothing
- [ ] Bugfixes
- [ ] Make a TaurSpriteAdapterComponent
- [ ] Program TaurSpriteAdapterComponent functionality

---

# Changelog

:cl:
- tweak: Changed minimum arachne age value from 60 to 18 and changed young age to 100
- tweak: Changed Anytaur inventory template to be a copy of default human template without feet slot
- tweak: Changed file structure of arachne textures to be more in line with how the normal file system works
- add: Added displacement map for jumpsuit slot.

JFerrix committed:
General UI fix

Fixes age limitations and changes the anytaur inventory template to be the same as human without feet slot

JFerrix committed:
Minor age update

JFerrix committed:
Added Displacementmap for jumpsuits

Adds a displacementmap for jumpsuits to not show legs anymore. Upside: Very simple and easy to further edit fix. Downside: Jumpsuits and jumpskirts now look basically identical. Oh well

JFerrix committed:
Update jumpsuit.png

JFerrix committed:
Delete jumpsuit.png.backup

KyuPolaris commented:
I have a PR with a fix for the inventory already involved, its mainly a slight shift to keep the look there but allow you to actually close it and have it keep its uniqueness <3

JFerrix committed:
Draft

Adds placeholder sprites, updates the code to recognise said sprites. Nonfunctional, for error fixing only
